src/test/java/com/mavenit/cucumber/PageObject/ResultsPage.java

```diff
@@ -131,14 +134,24 @@ public void selectPriceFilter(String priceOption) {
             }
         }
     }
-    public List<String> getPriceFIlter(){
-        List<String> priceList= new ArrayList<>();
-        for (WebElement price: priceSelection){
+
+    public List<String> getPriceFilter() {
+        List<String> priceList = new ArrayList<>();
+        for (WebElement price : priceSelection) {
             String priceTag = price.getText();
             priceList.add(priceTag);
         }
         return priceList;
     }
 
+    public void selectTwoFilters(String type, String price) {
+        selectTypeFilter(type);
+        selectPriceFilter(price);
+        driver.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
```

Comment:
Implicit wait here??? N y???

Reply:
I put the wait because the browser is not waiting till both filters have applied and then the rest of the test is failing.